src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv

```diff
@@ -0,0 +1,15 @@
+9.1.0,9.2.0,Transition notes
+0,0,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.

```

Comment:
The version numbers here don't match in line 1 and the number of variables needs to be specified in line 2.